client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java

```diff
@@ -100,24 +101,26 @@ public void testScroll() throws IOException {
         }
         {
             String scrollId = lastScrollId;
-
-            // tag::search-scroll-id
-            SearchScrollRequest scrollRequest = new SearchScrollRequest(); // <1>
-            scrollRequest.scroll(TimeValue.timeValueMinutes(1L)); // <2>
-            scrollRequest.scrollId(scrollId); // <3>
-            // end::search-scroll-id
-
             // tag::search-scroll-execute
-            SearchResponse searchResponse = client.searchScroll(scrollRequest); // <1>
-            SearchHits searchHits = searchResponse.getHits();  // <2>
-            String nextScrollId = searchResponse.getScrollId();  // <3>
+            while (true) {
```

Comment:
how about having a complete example with the initial search response too? Maybe leave a separate search scroll snippet, then move to the complete example where you reuse the SearchHits from the initial SearchResponse and you could do while(searchHits != null && searchHits.length > 0) instead of while(true)? Would that work or would it be too complicated?